.github/workflows/main.yml

```diff
@@ -33,3 +33,17 @@ jobs:
         run: pip install tox
       - name: Run tests using tox
         run: tox -e unit
+
+  integration-tests:
+    name: Integration tests
+    runs-on: ubuntu-latest
```

Comment:
Do we want to run integration tests in the latest ubuntu version?

Reply:
Fixed